docs/concepts/pools/boosted.md

```diff
@@ -6,73 +6,73 @@ order: 5
 
 ## Overview
 
-Boosted Pools are actually a subclass of [Composable Stable Pools](./composable-stable.md) but deserve their own page due to their powerful feature set. Boosted pools bring the best of both worlds to Liquidity Providers and Traders. Traders get access to deep stablecoin liquidity with near-parity exchange rates while Liquidity Providers get their liquidity positions sent to external protocols, such as [Aave](https://aave.com/).
+Boosted Pools are actually a subclass of other pools (typically [Composable Stable Pools](./composable-stable.md) and [Weighted Pools](./weighted.md)) but deserve their own page due to their powerful feature set. Boosted Pools bring the best of both worlds to Liquidity Providers (LPs) and Traders. Traders get access to deep liquidity with minimized price impact while Liquidity Providers get their liquidity positions sent to external protocols, such as [Aave](https://aave.com/).
 
-These versions of Stable Pools don't directly hold the stablecoins themselves, but rather hold the pool tokens (BPTs) of nested Linear Pools, which, for the Aave example, maintain proper balances of _TOKEN_ and _aTOKEN_.
+### Motivation
+In liquidity pools, swap prices are determined by pool balances. A large pool is great for traders: the larger the balances in the pool, the less the price changes from a given trade. The downside, however, is that large pools are not as great for LPs.
+While the full amount of the balances support token prices, only a fraction of the balances in the pool are actively traded back and forth to facilitate trades. LPs collect swap fees based on their proportional share of the pool, so as more and more people join the pool, a given LP's collected swap fees will become diluted.
+We therefore require a solution that makes use of these three facts:
+
+Large pools...
```

Comment:
The formatting of this title and list doesn't look great. I would either keep it in the paragraph or if it's important to highlight, lose the `Large Pools...` and bold and start the points with a capital letter

Reply:
I just scrapped the list since removing the dilution angle made it a list of two anyway